include/glow/Base/Type.h

```diff
@@ -36,6 +36,8 @@ using TypeRef = const Type *;
 
 constexpr unsigned max_tensor_dimensions = 6;
 
+using unsigned_t = uint32_t;
```

Comment:
can you comment here on when it would be appropriate to use this alias?

Reply:
Done.